docs/AboutUs.md

```
@@ -17,43 +17,38 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 [[homepage](https://andrewsoon.com)]
 [[github](https://github.com/andrewsoonqn)]
-[[portfolio](team/andrewsoon.md)]
+
+- Role: Developer
 
 ### Sayyid Mehdi
+
 <img src="images/lamemario.png" width="200px">
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)